README.md

```diff
@@ -40,8 +40,8 @@ database, run tests, and send your contribution.
 If you want to go further, you can:
 
 * Check out development roadmap on the [wiki](https://wiki.mozilla.org/Pontoon)
```

Comment:
Moving the roadmap off the wiki is a follow-up?

Reply:
Yes.